pom.xml

```diff
@@ -26,6 +26,7 @@
 		<wiremock.version>2.12.0</wiremock.version>
 		<cucumber.version>2.3.1</cucumber.version>
 		<restassured.version>3.0.3</restassured.version>
+		<start-class>io.github.dannyflowerz.calendaraggregator.CalendarAggregatorApplication</start-class>
```

Comment:
Is this really needed?

Reply:
Without this I get the following error during the `repackage` on project `calendar-aggregator-test`: `Execution default of goal org.springframework.boot:spring-boot-maven-plugin:1.5.9.RELEASE:repackage failed: Unable to find main class`